qualifier/src/main/scala/com/weshare/qualifier/DataContainers.scala

```diff
@@ -0,0 +1,46 @@
+package com.weshare.qualifier
+
+import com.fasterxml.jackson.databind.JsonNode
+
+trait DataContainers {
+  /**
+   * Put all types below
+   */
+  type Geo = (Double, Double)
```

Comment:
```LatLon``` would be more readable, I'm ok with the type alias, did you consider a case class since there would be no confusion about which coordinate is which?